File: Pointers/Pro4.cpp
// 4. Write a C program to input and print array elements using pointer.

#include <stdio.h>
int main() 
{
    int n,i;
    printf("Enter array size: ");
    scanf("%d", &n);
    int arr[n];
    printf("Enter %d array elements:\n", n);
    for (i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    printf("Array elements are:\n");
    int *ptr = arr;
    for (i = 0; i < n; i++) {
        printf("%d ", *(ptr + i));
    }
    return 0;
}

File: Assignment - 2_2nd Sem/Pro4.py
//4. 1+x+x^2+x^3+……………………………………+x^n

x=int(input("Enter the value of x: "))
n=int(input("Enter the value of n: "))
result=1
for i in range(1,n+1):
    result+=pow(x,i)
print(result)